src/main/python/hydro_sphere/vm_files/hydra_pkgs_install_slaves.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+sudo apt-get install -y python-dev python-pip libtool pkg-config build-essential autoconf automake
```

Comment:
Also for future work, probably makes sense to create functions for installing packages in helpers, that way same code can be used for both master and slave .